.travis.yml

```diff
@@ -0,0 +1,35 @@
+---
+sudo: required
+
+env:
```

Comment:
Instead of `env` you can use:

```
matrix:
  include:
    - os: linux
      dist: trusty
      sudo: required
    - os: osx
      osx_image: xcode7.2
```

https://docs.travis-ci.com/user/multi-os/

Reply:
I don't want multiple Travis OS's though, I want a matrix of values to use to generate docker containers of specific combinations. 

FWIW, the basic approach I'm using is from a TravisCI blog post:

https://blog.travis-ci.com/2017-11-30-testing-ansible-roles-using-docker-on-travis

Reply:
Well, if that's the case, then looks like this is the best available method.